manifest/v1alpha/service/validation.go

```diff
@@ -12,6 +16,55 @@ func validate(s Service) *v1alpha.ObjectError {
 	return v1alpha.ValidateObject(validator, s, manifest.KindService)
 }
 
+// atLeastDailyContinuousFreq validates that RRULE frequency is DAILY or higher (WEEKLY, MONTHLY, YEARLY) and
+// that it has continuous occurrences.
+var atLeastDailyContinuousFreq = govy.NewRule(func(rule *rrule.RRule) error {
+	if rule == nil {
+		return nil
+	}
+
+	if rule.Options.Count > 0 || !rule.Options.Until.IsZero() {
+		return errors.New("rrule must have continuous occurrences")
+	}
+
+	if rule.Options.Interval < 1 {
+		return errors.New("interval must be at least 1")
+	}
+
+	// Only allow daily, weekly, monthly, and yearly frequencies
+	allowedFrequencies := []rrule.Frequency{rrule.DAILY, rrule.WEEKLY, rrule.MONTHLY, rrule.YEARLY}
+	isAllowed := false
+	for _, freq := range allowedFrequencies {
+		if rule.Options.Freq == freq {
+			isAllowed = true
+			break
+		}
+	}
```

Comment:
This loop could be simplified by `slices.Contains` or by simple `if rule.Options.Freq > rrule.DAILY`